### DISCUSSION TITLE
After updating to 2.21 I am getting "CS9058	Feature 'ref readonly parameters' is not available in C# 11.0. Please use language version 12.0 or greater"

### DISCUSSION THREAD

**Discussion by Arugin** [dotnet/Silk.NET#2239] [Q&A] [answered]
Visual Studio 2022 10.3
As far as I understand this is happening because of this change: https://github.com/dotnet/Silk.NET/pull/1882
Does it mean that I have to start using c# 12 or am I missing something?

**Comment by Perksey** [comment, marked as answer]
Yes.
